src/WikiService.Validation/Rubric/EditRubricRequestValidator.cs

```diff
@@ -0,0 +1,154 @@
+﻿using FluentValidation;
+using FluentValidation.Validators;
+using LT.DigitalOffice.Kernel.Validators;
+using LT.DigitalOffice.WikiService.Data.Interfaces;
+using LT.DigitalOffice.WikiService.Models.Db;
+using LT.DigitalOffice.WikiService.Models.Dto.Requests.Rubric;
+using LT.DigitalOffice.WikiService.Validation.Rubric.Interfaces;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.WikiService.Validation.Rubric
+{
+  public class EditRubricRequestValidator : ExtendedEditRequestValidator<DbRubric, EditRubricRequest>, IEditRubricRequestValidator
+  {
+    private readonly IRubricRepository _rubricRepository;
+
+    private async Task HandleInternalPropertyValidationAsync(
+      Operation<EditRubricRequest> requestedOperation,
+      CustomContext context)
+    {
+      Context = context;
+      RequestedOperation = requestedOperation;
+
+      #region paths
+
+      AddСorrectPaths(
+        new List<string>
+        {
+          nameof(EditRubricRequest.Name),
+          nameof(EditRubricRequest.ParentId),
+          nameof(EditRubricRequest.IsActive)
+        });
+
+      AddСorrectOperations(nameof(EditRubricRequest.Name), new() { OperationType.Replace });
+      AddСorrectOperations(nameof(EditRubricRequest.ParentId), new() { OperationType.Replace });
+      AddСorrectOperations(nameof(EditRubricRequest.IsActive), new() { OperationType.Replace });
+
+      #endregion
+
+      #region Name
+
+      AddFailureForPropertyIf(
+        nameof(EditRubricRequest.Name),
+        x => x == OperationType.Replace,
+        new()
+        {
+          {
+            x => !string.IsNullOrWhiteSpace(x.value?.ToString()),
+            "Name must not be empty."
+          },
+          {
+            x => x.value.ToString().Trim().Length <= 150,
+            "Name is too long."
+          }
+        }, CascadeMode.Stop);
+
+      #endregion
+
+      #region IsActive
+
+      AddFailureForPropertyIf(
+       nameof(EditRubricRequest.IsActive),
+       x => x == OperationType.Replace,
+       new()
+       {
+         {
+           x => bool.TryParse(x.value?.ToString(), out bool _), "Incorrect rubric is active format"
+         },
+       });
+
+      #endregion
+
+      #region ParentId
+
+      await AddFailureForPropertyIfAsync(
+        nameof(EditRubricRequest.ParentId),
+        x => x == OperationType.Replace,
+        new()
+        {
+          {
+            async (x) =>
+            {
+              if (x.value?.ToString() is null)
+              {
+                return true;
+              }
+
+              if (!Guid.TryParse(x.value.ToString(), out Guid parentId))
+              {
+                return false;
+              }
+
+              return await _rubricRepository.DoesExistAsync(parentId);
```

Comment:
it can be shorter (lines 91-96):
Guid.TryParse(x.value.ToString(), out Guid parentId)
? await _rubricRepository.DoesExistAsync(parentId)
: false

or even lines 84-98 (I'm not sure - may be need some more parenthesis):
async x => x.value?.ToString() is null
  ? true
  : Guid.TryParse(x.value.ToString(), out Guid parentId)
      ? await _rubricRepository.DoesExistAsync(parentId)
      : false,
"Parent id doesn`t exist."

Reply:
first is good, thanks!
I think that if with ternary operator inside is much easier to understand than nested ternary operator